packages/edit-site/src/components/sidebar-dataviews/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalItemGroup as ItemGroup } from '@wordpress/components';
+import { page, columns } from '@wordpress/icons';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import { useLink } from '../routes/link';
+import { default as DEFAULT_VIEWS } from '../dataviews/default-views';
+import { unlock } from '../../lock-unlock';
+const { useLocation } = unlock( routerPrivateApis );
+import SidebarNavigationItem from '../sidebar-navigation-item';
+
+function DataViewItem( { dataview, isActive } ) {
+	const {
+		params: { path },
+	} = useLocation();
+	let icon;
+	switch ( dataview.view.type ) {
+		case 'list':
+			icon = page;
+			break;
+		case 'grid':
+			icon = columns;
+			break;
+	}
+	const linkInfo = useLink( {
+		path,
+		currentView: dataview.slug,
+	} );
+	return (
+		<SidebarNavigationItem
+			icon={ icon }
+			{ ...linkInfo }
+			aria-current={ isActive ? 'true' : undefined }
+		>
+			{ dataview.title }
+		</SidebarNavigationItem>
+	);
+}
+
+const PATH_TO_DATAVIEW_TYPE = {
+	'/pages': 'page',
+};
+
+export default function DataViewsSidebarContent() {
+	const {
+		params: { path, currentView = 'all' },
+	} = useLocation();
+	const viewType = PATH_TO_DATAVIEW_TYPE[ path ];
+
+	return (
+		<ItemGroup>
+			{ DEFAULT_VIEWS[ viewType ].map( ( dataview ) => {
```

Comment:
Related to my other comment, I think we could pass the views here instead of keeping `DEFAULT_VIEWS` in a separate file for all the views we're going to have. I'd love some input from others for this.

Reply:
I agree with you that they are specific to "pages" and we don't need that file in the "data views" folder but these default views are useful in both the sidebar and the frame, so we should have it in one of these two folders. I guess.

Reply:
The could be in `page-pages`, exported and passed as param in `<NavigatorScreen path="/pages">`. If we gather all default views in a different file than the handling of them feels a bit `detached` to me. No strong opinions here.

Reply:
A different thing here from the above discussion, is that if we try to go to single page from the list(ex. click the title), path is not defined and breaks the editor.